ethers-solc/src/config.rs

```diff
@@ -266,6 +266,15 @@ impl ProjectPathsConfig {
                         }
                     }
                 }
+                // if still unable to resolve, attempt to find the absolute import
+                // inside the project paths that's possible due to `--include-dir`
+                for path in [&self.sources, &self.tests, &self.scripts] {
```

Comment:
looping over `include_paths` should be equivalent here, this also has the benefit that we don't add `sources`, `tests` `scripts` as hardcoded include-paths

Reply:
i'm sus, because `include_paths` are a mutable ref here and get modified on every invocation

Reply:
I see, good point,

I think this is fine then, the worst that can happen is that the graph resolves a file but solc can't in which case we still fail as expected.